sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebViewProvider.java

```diff
@@ -0,0 +1,21 @@
+package com.hcaptcha.sdk;
+
+import android.webkit.WebView;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.FragmentActivity;
+
+import com.hcaptcha.sdk.tasks.OnFailureListener;
+import com.hcaptcha.sdk.tasks.OnLoadedListener;
+import com.hcaptcha.sdk.tasks.OnOpenListener;
+import com.hcaptcha.sdk.tasks.OnSuccessListener;
+
+public interface HCaptchaWebViewProvider extends
+        OnLoadedListener,
+        OnOpenListener,
+        OnSuccessListener<HCaptchaTokenResponse>,
+        OnFailureListener {
+    @NonNull
+    WebView getWebView();
```

Comment:
Besides webview, it can contain the other shared variables such as config, listener, etc.

Reply:
Yes we can but those shared variables we don't 'expose' to other classes

Reply:
Can't they have a `protected` modifier?